pkg/maps/configmap/cell.go

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package configmap
+
+import (
+	"fmt"
+
+	"github.com/cilium/cilium/pkg/hive"
+	"github.com/cilium/cilium/pkg/hive/cell"
+)
+
+// Cell initializes and manages the config map.
+var Cell = cell.Module(
+	"config-map",
+	"eBPF map config contains runtime configuration state for the Cilium datapath",
+
+	cell.Provide(newMap),
+	cell.Invoke(func(Map) {}),
```

Comment:
Forced instantiation really required?

Reply:
no not really - having the utime sync depending on it - which triggers the instantiation.

but i thought it would be best to instantiate the map properly so the datapath can rely on it. otherwise it kind of always needs someone who depends on it to get initialized. or does datapath code create the map as well and would work without this initialization from userspace?

Reply:
It looks to me that we do not create any bpf maps from the datapath?

Reply:
introduced a bpf registry where map cells register themselves by using the value group `bpfMaps`.

this way datapath constructor can depend on the bpf map registry and the initialization order is ensured.

it's just a start which addresses this issue. might be enriched / replaced in the future.

@joamaki can you take another look :eyes: